### PR TITLE
Is it boolean.js

### PR DESCRIPTION
**Pull request by DGEs2018**
@yhabib @aiwatko could you please review this ? Thank you !

### CODE REVIEW

File: challenges/is-it-Boolean.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+function isItBoolean(value) {
+	if (value === true || value === false) {
+		console.log(True + ", it's indeed a Boolean");
```
Comment (aiwatko) [inline challenges/is-it-Boolean.js]: this line will throw the following error when you try to call your function: `Uncaught ReferenceError: True is not defined`. I will let you discover why it is to, let me know if you need help :)

Comment (DGEs2018) [reply challenges/is-it-Boolean.js]: Thanks for spotting that @aiwatko. It's now fixed, you might want to review that now

File: challenges/is-it-Boolean.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+/*
+Write a function that checks if a given value is Boolean or not and returns true or false.
+*/
+function isItBoolean(value) {
+	if (value === true || value === false) {
+		return true;
```
Comment (aiwatko) [inline challenges/is-it-Boolean.js]: in both cases we want to return a string, not a boolean value :)

Comment (DGEs2018) [reply challenges/is-it-Boolean.js]: I just fixed and re-pushed the as discussed - thank you again for the lesson !

**Pull request merged**